auth.mkd

```diff
@@ -757,45 +746,45 @@ function decode_check(auth_pages[], decoded_lpi, decoded_length) {
 
 ## Operational {#operational-recommendations}
 
 UAS operation may impact the frequency of sending DRIP Authentication messages. When a UA dwells at an approximate location, and the channel is heavily used by other devices, less frequent message authentication may be sufficient for an Observer. Contrast this with a UA traversing an area, where every message should be authenticated as soon as possible for greatest success as viewed by the receiver.
 
-Thus how/when these DRIP Authentication Messages are sent is up to each implementation. Further complication comes in comparing Legacy and Extended Transports.  In Legacy, each message is a separate hash within the Manifest. So, when the UA remains within a small volume of airspace, it may lean toward occasional message authentication. In Extended Transports, the hash is over the Message Pack so only few hashes need to be in a Manifest. A single Manifest can handle a potential two Message Packs (for a full set of messages) and a DRIP Link Authentication Message for the `Broadcast Endorsement: HDA, UA`.
+Thus how/when these DRIP Authentication Messages are sent is up to each implementation. Further complication comes in comparing Legacy and Extended Transports.  In Legacy, each message is a separate hash within the Manifest. So, when the UA remains within a small volume of airspace, it may lean toward occasional message authentication. In Extended Transports, the hash is over the Message Pack so only few hashes need to be in a Manifest. A single Manifest can handle a potential two Message Packs (for a full set of messages) and a DRIP Link Authentication Message for the `BE: HDA, UA`.
 
-A separate issue is the frequency of transmitting the DRIP Link Authentication Message for the `Broadcast Endorsement: HDA, UA` when using the Manifest. This message content is static; its hash never changes radically. The only change is the 4-byte timestamp in the Authentication Message headers. Thus, potentially, in a dwell style operation it can be sent once per minute, where its hash is in every Manifest. A receiver can cache all DRIP Link Authentication Message for the `Broadcast Endorsement: HDA, UA` to mitigate potential packet loss.
+A separate issue is the frequency of transmitting the DRIP Link Authentication Message for the `BE: HDA, UA` when using the Manifest. This message content is static; its hash never changes radically. The only change is the 4 octet timestamp in the Authentication Message headers. Thus, potentially, in a dwell style operation it can be sent once per minute, where its hash is in every Manifest. A receiver can cache all DRIP Link Authentication Message for the `BE: HDA, UA` to mitigate potential packet loss.
```

Comment:
```suggestion
A separate issue is the frequency of transmitting the DRIP Link Authentication Message for the `BE: HDA, UA` when using the Manifest. This message content is static; its hash never changes radically. The only change is the 4-octet timestamp in the Authentication Message headers. Thus, potentially, in a dwell style operation it can be sent once per minute, where its hash is in every Manifest. A receiver can cache all DRIP Link Authentication Message for the `BE: HDA, UA` to mitigate potential packet loss.
```